packages/fast-documentation/src/footer/footer.style.ts

```diff
@@ -0,0 +1,89 @@
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { FooterClassNameContract } from "./footer.props";
+import { DesignSystem } from "../design-system/index";
+import {
+    applyFontWeightSemiBold,
+    applyScaledTypeRamp,
+    neutralForegroundHint
+} from "@microsoft/fast-components-styles-msft";
+
+const styles: ComponentStyles<FooterClassNameContract, DesignSystem> = {
+    "@global": {
+        body: {
+            fontFamily: "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif",
+            margin: "0",
+            padding: "0"
+        }
+    },
```

Comment:
Could this be moved to a stylesheet for the app or whatnot? It's not directly related to the footer.